sdk/core/Azure.Core/samples/Response.md

```diff
@@ -127,6 +127,51 @@ await foreach (Page<SecretProperties> page in allSecretProperties.AsPages())
 }
 ```
 
+## Using System.Linq.Async with AsyncPageable
+
+The [`System.Linq.Async`](https://www.nuget.org/packages/System.Linq.Async) package provides a set of LINQ methods that operate on `IAsyncEnumerable<T>` type.
```

Comment:
I was going to suggest we link to the docs...   but https://github.com/dotnet/reactive/issues/1097.

+@jongio - maybe we could use our vendor team to get the `System.Linq.Async` docs published since we're recommending that to customers using `IAsyncEnumerable`?

Reply:
I have a thread with System.Linq.Async owners about the docs.